ironfish-cli/src/commands/accounts/pay.ts

```diff
@@ -195,19 +195,26 @@ ${displayIronAmountWithCurrency(
 
     try {
       const result = await client.sendTransaction({
-        amount: ironToOre(amount).toString(),
         fromAccountName: from,
-        memo: memo,
-        toPublicKey: to,
+        receives: [
+          {
+            publicAddress: to,
+            amount: ironToOre(amount).toString(),
+            memo: memo,
+          },
+        ],
         fee: ironToOre(fee).toString(),
         expirationSequence,
       })
 
       stopProgressBar()
 
       const transaction = result.content
+      const recipients = transaction.receives
+        ?.map((receive) => receive.publicAddress)
```

Comment:
Can we delete the `?` here? I'm having trouble looking for the origin type to see if `transaction.receives` can ever be undefined/null